IdentityCore/src/parameters/MSIDInteractiveRequestParameters.h

```diff
@@ -42,4 +42,5 @@
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->